### ISSUE TITLE
Add OVH DNS provider

### ISSUE THREAD

**Issue by bcyrill** [janeczku/rancher-letsencrypt#17]
Support for OVH has recently been integrated in lego (https://github.com/xenolf/lego/pull/233). I'd be happy to test it.

**Comment by bcyrill** [comment]
Has been implemented in the following pull request #25

**Comment by gelnior** [comment]
When do you plan to release this change? In the Rancher catalog, it's not available. 

By the way, thank you for that service which makes cert management easy!

**Comment by disaster37** [comment]
While there are no new docker image, you can use this : quay.io/webcenter/rancher-letsencrypt:3.0.1

It's build with the last master merge and contain OVH plugin (I use this image with Ovh dns provider)

**Comment by gelnior** [comment]
Ok thank you @disaster37

**Comment by ebreton** [comment]
Hi guys, 

I am trying rancher and I would like to migrate my current installation into it. It has a [https-portal](https://github.com/SteveLTN/https-portal) set up, which makes use of letsencrypt.

My servers are hosted @ovh, and I am interested in @disaster37 solution but I don't have enough understanding of rancher to make it work. I have signed up on quay.io, I tried to make it a new registry on rancher... but I am kind of stuck here. 

How should I do ?
Cheers,
Manu
